requirements/extras.txt

```diff
@@ -0,0 +1,5 @@
+ovos-backend-client>=0.1.0a7
+ovos-microphone-plugin-alsa~=0.0.0
+ovos-stt-plugin-server~=0.0.3
+ovos-ww-plugin-precise-lite[full]~=0.1
```

Comment:
Should not use full, if anything use lite

Reply:
my thinking here is `lite` isn't compatible with Python3.11 but I don't have any strong feelings about the defaults since I won't generally install any of them